pages/linux/synopkg.md

```diff
@@ -0,0 +1,36 @@
+# synopkg
+
+> Package management utility for Synology DiskStation Manager.
+> More information: <https://www.synology.com/dsm>.
+
+- List installed packages without description:
+
+`synopkg list --name`
+
+- List packages depend on a specific package:
+
+`synopkg list --depend-on {{package}}`
+
+- Start/Stop a package:
+
+`sudo synopkg {{start|stop}} {{package}}`
+
+- Print the status of a package:
+
+`synopkg status {{package}}`
+
+- Uninstall a package:
+
+`sudo synopkg uninstall {{package}}`
+
+- Check if a package is updateable:
+
+`synopkg checkupdate {{package}}`
+
+- Upgrade all packages to latest version:
+
+`sudo synopkg upgradeall`
+
+- Install a package from file:
```

Comment:
```suggestion
- Install a package from a `synopkg` file:
```
```suggestion
- Install a package from a synopkg file:
```

Reply:
The second suggestion from @CleanMachine1 works best I think. The enclosing backticks "`" are not necessary in this instance I don't think.